configuration/developing-community-health-applications.md

```diff
@@ -388,6 +391,22 @@ So why not put everything in `resolvedIf` and this will always work without thin
 
 Logic to test if a contact has a form x existing before any form y, or testing if the task's action form is present within the event window - these are examples of bad things to put in `appliesIf` functions.
 
+##### Core Framework 3.8.0 and Later
+**Declarative configuration projects must deploy their configuration using medic-conf v3.1.0 or later for tasks and targets to function on the Core Framework v3.8.0 or later**
+
+In the Core Framework 3.8.0 release, the task system is powered by task documents which are written to PoucHDB and synced to the CouchDB/Postgres servers. This powers collaborative features, and enables Postgres impact queries to understand how tasks behave on phones in the field. The behavior described above (Before Core Framework 3.8.0) is no longer true, tasks will disappear whenever `appliesIf: false`. Instead `resolvedIf` now controls when task documents move into the "Completed" state vs "Cancelled" state.
```

Comment:
Some minor grammatical changes, and also clarifying what happens if appliesIf is false, as per last point in https://github.com/medic/cht-core/issues/5657#issuecomment-559848482

```suggestion
In the Core Framework 3.8.0 release, the task system is powered by task documents which are written to PouchDB and then synced to CouchDB and Postgres. This powers collaborative features, and enables Postgres impact queries to understand how tasks behave on phones in the field.

 The behavior described above (Before Core Framework 3.8.0) is no longer true, as tasks will be given the "Cancelled" state and disappear whenever `appliesIf: false`. Instead `resolvedIf` now controls when task documents move into the "Completed" state vs "Cancelled" state.
```